modules/minimp3/audio_stream_mp3.cpp

```diff
@@ -133,7 +133,10 @@ void AudioStreamPlaybackMP3::seek(double p_time) {
 		return;
 	}
 
-	if (p_time >= mp3_stream->get_length()) {
+	double max = mp3_stream->get_length();
+	if (p_time < 0) {
```

Comment:
Just one if-clause. Both clause bodies do the same…

```suggestion
	if (p_time < 0 || p_time >= mp3_stream->get_length()) {
		p_time = 0;
	}
```